docker/entrypoint_prod.sh

```diff
@@ -32,7 +32,7 @@ elif [ "$1" == "pulse_listener_tasks" ]; then
 # Processes pushes/jobs from Pulse that were collected by `pulse_listener_{pushes,tasks}`.
 elif [ "$1" == "worker_store_pulse_data" ]; then
     export REMAP_SIGTERM=SIGQUIT
-    exec newrelic-admin run-program celery worker -A treeherder --without-gossip --without-mingle --without-heartbeat -Q store_pulse_pushes,store_pulse_tasks --concurrency=3
+    exec newrelic-admin run-program celery worker -A treeherder --without-gossip --without-mingle --without-heartbeat -Q store_pulse_pushes,store_pulse_tasks,store_pulse_tasks_classification --concurrency=3
```

Comment:
hrm, is the store_pulse_tasks_classification supposed to be listening to the firefox-ci or community cluster? `pulse_listener_pushes|tasks` is listening to firefox-ci: https://github.com/mozilla/treeherder/blob/master/treeherder/etl/management/commands/pulse_listener_pushes.py#L31

I think you might need to split `store_pulse_tasks_classification` out into its own worker to listen to community cluster, and then @jmaher will need to add an entry here: https://github.com/mozilla-services/cloudops-infra/blob/7249a5752ebefcdd645e970a1ef6459ea3b8b42f/projects/treeherder/k8s/charts/treeherder/values.yaml#L75-L155

Reply:
And per my comment below, its only listening for pushes to defined projects in PROJECTS_TO_INGEST env variable.

Reply:
`store_pulse_tasks_classification` is supposed to react to messages from the community cluster, hence the new cluster configuration I added in the command `PULSE_SOURCES` attribute

I was able to make this work without splitting it to another worker. Since all three queues refer to storing data from Pulse messages, I thought it was good to put it in the same worker.

The worker don't actively listen to pulse messages, it's the Django command that do. The worker is only here to execute asynchronous tasks triggered after the reception of a Pulse message.
By using the `pulse_listener` Django command, we are listening to the firefox-ci cluster, using the first source from `PULSE_SOURCES`, we should never trigger `store_pulse_tasks_classification` tasks are they are only running on community.
And **at the same time**, we are also listening to the community cluster, using the second source from `PULSE_SOURCES` where we have defined special bindings, that will launch `store_pulse_tasks_classification` tasks.

Reply:
If you got it working locally, great! Please keep in mind my questions were designed to help figure out why it wasn't working on prototype. :)

Reply:
Yes sure! Sorry if I sounded a little abrupt that wasn't the goal. And I would be happy to refactor anything that doesn't match the right way of doing things in Treeherder, I'm new to the codebase so I might have missed some generic rules to follow. I was just explaining my reasoning :smile: 

As I said in the comments below, the fact that it isn't working on proptotype might come from the lack of rights on the Pulse Guardian user